python/cudaq/kernel/ast_bridge.py

```diff
@@ -246,11 +246,26 @@ def getFloatAttr(self, type, value):
     def getConstantFloat(self, value):
         """
         Create a constant float operation and return its MLIR result Value.
-        Takes as input the concrete float value. 
+        Takes as input the concrete float value.
         """
         ty = self.getFloatType()
         return arith.ConstantOp(ty, self.getFloatAttr(ty, value)).result
 
+    def getComplexType(self):
+        """
+        Return an MLIR complex type (double precision).
+        """
+        return ComplexType.get(self.getFloatType())
```

Comment:
Q: Is there a way to request either F32 or F64 in the python bindings? (This getFloatType() feels more ambiguous.)